site/pages/migration-guides/migrating-to-v3.md

```diff
@@ -20,7 +20,17 @@ head:
 
 # Migration Guide
 
-Below are the steps to migrate your project from older versions of the `aa-sdk` to the latest version.
+Below are the steps to migrate your project from older versions of the `aa-sdk` and `account-kit` to the latest version.
+
+## Migrating to version v4.x.x
```

Comment:
Note: this is no where near final. just wanted to catch these things before I forgot about them